airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -802,20 +1281,33 @@ definitions:
         type: string
         enum: [ListPartitionRouter]
       cursor_field:
+        title: Cursor Field
```

Comment:
This is called `Current partition value identifier` in the UI which seems better

Reply:
updated